dbt/models/default/default.vw_pin_appeal.sql

```diff
@@ -1,10 +1,19 @@
 -- View containing appeals by PIN
 
--- This CTE exists only so that we can join reason descriptions onto cleaned
--- reason codes.
+-- CTE so that we can join reason descriptions onto cleaned reason codes.
 WITH reasons AS (
-    SELECT
-        htpar.*,
+    SELECT DISTINCT
+        htpar.parid,
+        htpar.taxyr,
+        htpar.caseno,
+        htpar.user38,
+        htpar.user104,
+        htpar.resact,
+        htpar.hrstatus,
+        htpar.cpatty,
+        -- user125 isn't used in the view, but it's FilingID and we want to
+        -- include it to make sure we're only dropping true dupes using DISTINCT
+        htpar.user125,
```

Comment:
Because I'm using `DISTINCT` here I decided to apply it to the smallest number of columns possible in the CTE